### PR TITLE
Wire edgeless in simulation mode for local testnet

### PR DESCRIPTION
**Pull request by tudor-malene**
### Why this change is needed

Sqlite doesn't work properly under load. The e2e tests running against a "local testnet" deployment fail with "database locked"

### What changes were made as part of this PR

replace sqlite with edgeless for the "local testnet"

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/ten-protocol/ten-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [x] PR checks reviewed and performed

### CODE REVIEW

File: go/node/docker_node.go [hunk 1]
```diff
@@ -204,25 +200,27 @@ func (d *DockerNode) startEnclave() error {
...
+	} else {
+		envs["OE_SIMULATION"] = "1"
 	}
 
 	// only set the pccsAddr env var if it's defined
 	if d.cfg.pccsAddr != "" {
 		envs["PCCS_ADDR"] = d.cfg.pccsAddr
 	}
 
+	// todo - do we need this volume?
```
Comment (tudor-malene) [inline go/node/docker_node.go:220]: @BedrockSquirrel , can you think of any situation where we need the data between different runs of the testnet? 
(Other than upgrades, which we don't test locally currently?)

Comment (BedrockSquirrel) [reply go/node/docker_node.go:222]: I'm a bit confused, yeah it's just for upgrades. But doesn't this code gets used for the main testnets as well as the local testnets?

Comment (BedrockSquirrel) [reply go/node/docker_node.go:221]: Worth deleting this then, or want to leave it in case we need to add it back in?

**Review by BedrockSquirrel — APPROVED** [review]
LGTM

File: go/node/docker_node.go (hunk 1 above)
Comment (tudor-malene) [reply go/node/docker_node.go:221]: yeah. there might be scenarios we don't know

**Pull request merged**